node/src/node_configurator/unprivileged_parse_args_configuration.rs

```diff
@@ -1972,6 +1988,52 @@ mod tests {
         //no prepared results for the setter methods, that is they're uncalled
     }
 
+    #[test]
+    fn configure_accountant_config_discovers_invalid_combination_in_payment_thresholds_params_from_user_input(
+    ) {
+        let multi_config = make_simplified_multi_config([
+            "--payment-thresholds",
+            "19999|10000|1000|20000|1000|20000",
+        ]);
+        let mut bootstrapper_config = BootstrapperConfig::new();
+        let mut persistent_config =
+            configure_default_persistent_config(ACCOUNTANT_CONFIG_PARAMS | MAPPING_PROTOCOL)
+                .set_payment_thresholds_result(Ok(()));
+
+        let result = configure_accountant_config(
+            &multi_config,
+            &mut bootstrapper_config,
+            &mut persistent_config,
+        );
+
+        let expected_msg = "Value of DebtThresholdGwei (19999) must be bigger than of PermanentDebtAllowedGwei (20000)";
```

Comment:
Maybe this error message could be improved. Either take out the "of" or precede it with "the value."